gateways/common/webhook.go

```diff
@@ -224,3 +226,24 @@ func ProcessRoute(rc *RouteConfig, helper *WebhookHelper, eventStream gateways.E
 	}
 	return nil
 }
+
+func ValidateWebhook(endpoint, port string) error {
+	if endpoint == "" {
+		return fmt.Errorf("endpoint can't be empty")
+	}
+	if port == "" {
+		return fmt.Errorf("port can't be empty")
+	}
+
+	if !strings.HasPrefix(endpoint, "/") {
```

Comment:
do you think this is a fatal error? can't we easily handle this case by forming a correct endpoint by prepending a slash if necessary?

Reply:
there also might be standard libraries that parse both `endpoint` and `port` together to form an address.. you can look at leveraging those.